<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendInfo</class>
 <widget class="QWidget" name="FriendInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>65</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>65</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>270</width>
    <height>65</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*全局样式表*/
QWidget#FriendInfo{
	background-color: rgb(255, 255, 255);
	border-bottom:2px solid rgb(149, 149, 149);
}

/*局部样式表*/
QLabel#userHead_Lab{
	image: url(:/new/user/user/touxiang01.jpg);
	border:1px solid rgb(0, 255, 0);
}

</string>
  </property>
  <widget class="QLabel" name="userHead_Lab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="userName_Lab">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="userSign_Lab">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>35</y>
     <width>170</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>170</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>170</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>个性签名</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
